rega-genotype/src/main/java/rega/genotype/GenotypeTool.java

```diff
@@ -317,11 +318,12 @@ public void setParent(GenotypeTool parent) {
     
 	public static void main(String[] args)
     	throws IOException, ParameterProblemException, FileFormatException, ClassNotFoundException, InstantiationException, IllegalAccessException, IllegalArgumentException, SecurityException, InvocationTargetException, NoSuchMethodException {
-    	
+		
     	ArgsParseResult parseArgsResult = parseArgs(args);
+    	//System.out.println(parseArgsResult);
     	if (parseArgsResult.remainingArgs == null)
     		return;
-
+    	
```

Comment:
Please remove all the changes to the whitespaces that have no effect on this file